src/coreclr/vm/debuginfostore.h

```diff
@@ -177,6 +204,6 @@ class DebugInfoManager
 #endif
 };
 
-#define DebugInfoBoundsHasInstrumentedBounds 0xFFFFFFFF
+#define DebugInfoFat 0
```

Comment:
Switched this to 0 since encoding `0xFFFFFFFF` takes up a full 6 bytes, and the value 0 is not a common value for `cbBounds` (JIT never reports empty bounds over all 1+ million methods in our SPMI collections, I suspect we always have at minimum a prolog bound)

Reply:
Although I am somewhat worried it is possible to see this from the R2R path. I wonder if we should update R2R major version and teach crossgen2 to produce the fat header if necessary. We'll probably need to encode the debug info in R2R anyway during .NET 11, so I guess we might as well take that update now.

Reply:
@davidwrighton Any thoughts on this?

Reply:
Updating the R2R major version, and such is the thing to do.